src/App.tsx

```diff
@@ -538,18 +499,24 @@ function App(): ReactElement {
   timeControls.setFrameCallback(setFrame);
   recordingControls.setFrameCallback(setFrame);
 
-  const getImagePrefix = (): string => imagePrefix || `${datasetName}-${featureName}-`;
+  const getImagePrefix = (): string => imagePrefix || `${datasetKey}-${featureName}-`;
 
   // RENDERING /////////////////////////////////////////////////////////////
 
-  const [notificationApi, notificationContextHolder] = notification.useNotification();
+  const notificationConfig: NotificationConfig = {
+    getContainer: () => {
+      return notificationContainer.current as HTMLElement;
+    },
```

Comment:
nit: I prefer one-line arrow functions whenever the function's only job is to evaluate a one-line expression and return its value, like so:
```typescript
getContainer: () => notificationContainer.current as HTMLElement;
```
that's just me though